data-raw/standard-format-data-prep/adult_upstream_passage_standard_format.Rmd

```diff
@@ -59,21 +60,71 @@ Below we read in the adult upstream passage data for each monitoring program and
-#                overwrite = TRUE)
-
-battle_upstream_passage <- read_csv("../../data/adult-upstream-passage-monitoring/battle_adult_upstream.csv") %>% 
-  mutate(stream = "Battle Creek") %>% 
-  filter(count != 0) %>% # occurs when catch of non chinook fish
-  glimpse
+gcs_get_object(object_name = "adult-upstream-passage-monitoring/battle-creek/data/battle_passage_video.csv",
+               bucket = gcs_get_global_bucket(),
+               saveToDisk = here::here("data", "adult-upstream-passage-monitoring", "battle_adult_upstream.csv"),
+               overwrite = TRUE)
+
+battle_video <- read_csv(here::here("data", "adult-upstream-passage-monitoring", "battle_adult_upstream.csv")) %>% 
+  mutate(stream = "battle creek",
+         method = "video",
+         adipose_clipped = case_when(adipose == "present" ~ F, # originally adipose was present/absent which maps to F/T for clipped
+                             adipose == "absent" ~ T)) %>% 
+  filter(count != 0) %>% # occurs when catch of non chinook fish 
+  select(-adipose) |> 
+  glimpse()
+
+video_start_date <- battle_video |> 
+  group_by(year = year(date)) |> 
+  summarize(start_date = min(date))
+# battle_video |> 
+#   mutate(day = day(date),
+#          month = month(date),
+#          year = year(date),
+#          fake_date = as_date(paste0("2000-", month, "-", day))) |> 
+#   ggplot(aes(x = fake_date, y = count, color = year)) +
+#   geom_point()
+
+gcs_get_object(object_name = "adult-upstream-passage-monitoring/battle-creek/data/battle_passage_trap.csv",
+               bucket = gcs_get_global_bucket(),
+               saveToDisk = here::here("data", "adult-upstream-passage-monitoring", "battle_passage_trap.csv"),
+               overwrite = TRUE)
+
+battle_spawning_building <- read_csv(here::here("data", "adult-upstream-passage-monitoring", "battle_passage_trap.csv")) |> 
+  mutate(stream = "battle creek",
+         method = "trap",
+         time = as_hms(trap_beg),
+         adipose_clipped = case_when(adipose == "present" ~ F, # originally adipose was present/absent which maps to F/T for clipped
+                             adipose == "absent" ~ T)) 
+# battle_spawning_building |> 
+#   mutate(day = day(date),
+#          month = month(date),
+#          fake_date = as_date(paste0("2000-", month, "-", day))) |> 
+#   ggplot(aes(x = fake_date, y = count)) +
+#   geom_point()
+spawning_end_date <- battle_spawning_building |> 
+  group_by(year = year(date)) |> 
+  summarize(end_date = max(date))
+
+date_switch <- full_join(video_start_date, spawning_end_date)
+
+battle_upstream_passage <- battle_video |> 
+  bind_rows(battle_spawning_building |> 
+              select(date, stream, time, count, sex, confidence_in_sex, fork_length, adipose_clipped, status) |> 
+              mutate(dead = ifelse(status %in% c("sacraficed", "mortality", "dead in pond"), T, F)) |> # label mortalities
```

Comment:
do we want to add a `mutate` here that corrects the spelling of `sacraficed` to `sacrificed` ? something like 
`mutate(status = ifelse(status == "sacraficed", "sacrificed", status),
       dead = ifelse(status %in% c("sacrificed", "mortality", "dead in pond"), T, F))`

Reply:
thank you - fixed!